changelog.d/13446.removal

```diff
@@ -0,0 +1 @@
+Remove the unused `get_users` storage method.
```

Comment:
Not sure this is worth calling out. I don't think this is user-visible, either to end users or module api users(?).